### PR TITLE
Remove JSONP support

### PR DESCRIPTION
**Pull request by osma**
While I'm at it making PRs, I propose that we drop JSONP support altogether. This PR does it.

As already explained in #19, it's an obsolete an insecure mechanism for making cross-origin HTTP requests. It was already made optional for spec version 0.2, but I think it's better to leave it out from the API spec entirely because CORS is already mandated (since 0.2) and does the same thing, but in a better way.

### CODE REVIEW

**Review by wetneb — APPROVED** [review]
Yes, JSONP was only there for backwards compatibility, but since this new spec has breaking changes it does not make sense to keep it. Services are free to still implement version 0.1 or 0.2 of the specs on top.

**Comment by osma** [comment]
Thanks for the quick approval @wetneb!
Can I just merge this or do we need to wait for opinions from others?

**Comment by wetneb** [comment]
It seems relatively uncontentious but let's wait for another opinion, I guess it should not have merge conflicts with other open PRs anyway.

**Comment by osma** [comment]
The problem with merge conflicts is caused by the list of changes included in the spec. I thought it would make sense to add a brief mention of the changes I made in the same PR that implements those changes. But that inevitably leads to merge conflicts when there are more than one open PRs making that kind of changes. Fortunately those should be easy to resolve.

File: draft/index.html
Comment (tfmorris) [inline draft/index.html:221]:
```suggestion
            <li>The reconciliation queries only support passing a single <code>type</code> and the <code>type_strict</code> parameter was removed</li>
            <li>Remove JSONP support</li>
```
I'm not sure if there's a specific style guide being followed which requires this punctuation, but it seems odd to me. It's certainly not commonly used in American English. I would suggest removing all the terminal punctuation for the list entries. (Applies to the two list entries above as well)

**Review by tfmorris — CHANGES_REQUESTED** [review]
Looks good except for a minor nit with list punctuation (which wasn't introduced in this PR, but made noticeable because of it).

File: draft/index.html
Comment (osma) [reply draft/index.html:221]: Yeah, I noticed that as well, but I figured that it's best to just keep following the established style.

Should I remove the punctuation in this PR? Make another PR? There are two more lists that use similar punctuation above this one (the ones for versions 0.1 and 0.2).

Comment (wetneb) [reply draft/index.html:221]: Agreed, we can change the formatting of the list in another PR.

**Pull request merged**
